pkg/registry/servicecatalog/binding/storage.go

```diff
@@ -94,12 +94,12 @@ func toSelectableFields(binding *servicecatalog.Binding) fields.Set {
 }
 
 // GetAttrs returns labels and fields of a given object for filtering purposes.
-func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, error) {
+func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, bool, error) {
```

Comment:
> I do not know what that means. My guess from looking at the commit is whether the thing that we're getting has been set?

it checks whether there is still at least one initializer set (see comment to the field [meta/v1/types.go#L216](https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L216)):
```
	// This field is a list of initializers that have not yet acted on this object. If nil or empty,
	// this object has been completely initialized. Otherwise, the object is considered uninitialized
	// and is hidden (in list/watch and get calls) from clients that haven't explicitly asked to
	// observe uninitialized objects.
```
if there is at least one initializer set, it means that object is not fully initialized yet. Or, in other words, it is "uninitialized". The uninitialized objects are ignored for list/watch requests by default, as the comment above describes - that's what "unitialized flag" is for.